packages/debian/libgroonga0.install

```diff
@@ -1,16 +1,8 @@
 etc/groonga/synonyms.tsv
-# ggml/ doesn't exist with Ubuntu 20.04 becuase CMake is old.
-# We can split this when we drop support for Ubuntu 20.04.
-#
-# usr/lib/*/groonga/ggml/*
-# usr/lib/*/groonga/scripts/*
-usr/lib/*/groonga/[gs]*/*
-# language_model.* doesn't exist with Ubuntu 20.04 becuase CMake is old.
-# We can split this when we drop support for Ubuntu 20.04.
-#
-# usr/lib/*/groonga/plugins/functions/index_column.*
-# usr/lib/*/groonga/plugins/functions/language_model.*
-usr/lib/*/groonga/plugins/functions/[il]*.*
+usr/lib/*/groonga/ggml/*
+usr/lib/*/groonga/scripts/*
```

Comment:
Could you keep this list in alphabetical order?

Reply:
fix: [ffafd7f](https://github.com/groonga/groonga/pull/2218/commits/ffafd7f81ec54593133027d68c58db415d29989a) Sure.